packages/location-field/src/stories/Desktop.story.js

```diff
@@ -104,3 +108,30 @@ export const WithBadApiKeyHandlesBadAutocomplete = () => (
     onLocationSelected={onLocationSelected}
   />
 );
+
+export const WithCustomResultColorsAndIcons = () => (
+  <LocationField
+    currentPosition={currentPosition}
+    geocoderConfig={geocoderConfig}
+    getCurrentPosition={getCurrentPosition}
+    inputPlaceholder="Select from location"
+    locationType="from"
+    onLocationSelected={onLocationSelected}
+    layerColorMap={{ stops: "green", stations: "orange" }}
+    operatorIconMap={{ "wa-state-ferry": <Ship size={13} /> }}
+  />
+);
+
+export const WithPrefilledSearch = () => (
+  <LocationField
+    initialSearchResults={mockedGeocoderResponse.features}
+    layerColorMap={layerColorMap}
+    currentPosition={currentPosition}
+    geocoderConfig={geocoderConfig}
+    getCurrentPosition={getCurrentPosition}
+    inputPlaceholder="Select from location"
```

Comment:
Sort props.

Reply:
Sorted in e0fc8a811358ed8a55a587f948db1614da68dbd4